connaisseur/validators/notaryv1/notary.py

```diff
@@ -89,39 +89,56 @@ def healthy(self):
         except Exception:
             return False
 
-    async def get_trust_data(
-        self,
-        session: aiohttp.ClientSession,
-        image: Image,
-        role: TUFRole,
-        token: str = None,
+    async def get_auth_and_root_trust_data(
```

Comment:
Since this breaks what one would usually program regarding modularity, I'd say a comment would be beneficical here

Reply:
And since you return a tuple, which is always harder to understand, supportive type annotations for the output would be nice